apps/nova/src/hooks/call.ts

```diff
@@ -357,3 +357,60 @@ export function useCallEvents(
     mateVideo,
   };
 }
+
+export function useFullScreen() {
+  const intl = useFormatMessage();
+  const ref = useRef<HTMLDivElement>(null);
+  const [isFullScreen, setIsFullScreen] = useState<boolean>(false);
+
+  const onFullScreen = useCallback(() => {
+    const fullScreen = document.fullscreenElement?.id === ref.current?.id;
+    setIsFullScreen(fullScreen);
+  }, []);
+
+  const startFullScreen = useCallback(async () => {
+    try {
+      if (!ref.current) return;
+      await ref.current.requestFullscreen();
+    } catch (error) {
+      toaster.error({
+        title: intl("error.unexpected"),
+      });
+    }
```

Comment:
Use our `safe` util from `@litespace/sol`

```suggestion
  const result = await safe(async () => {
      if (!ref.current) return;
      await ref.current.requestFullscreen();
   })
   if (result instanceof Error) toaster.error({ title: intl("error.unexpected") });
```